src/components/Nav/Nav.js

```diff
@@ -58,12 +62,17 @@ const Nav = () => {
             onClick={() => navigate('/cart')}
           />
           <button
-            className="login-btn"
+            className={tokenValue === null ? 'login-btn' : 'display-none'}
```

Comment:
```suggestion
            className={tokenValue ? 'login-btn' : 'display-none'}
```
tokenValue가 없으면 자동으로 false 값처럼 여겨져서 이렇게 구현해도 됩니다!